code/modules/mob/mob_movement.dm

```diff
@@ -215,6 +215,9 @@
 ///Called by client/Move()
 ///Checks to see if you are being grabbed and if so attemps to break it
 /client/proc/Process_Grab()
+	if(mob.incapacitated()) // Can't break out of grabs if you're incapacitated
```

Comment:
The incapacitated check also takes a parameter to ignore grabs. But this does nothing